src/twisted/internet/testing.py

```diff
@@ -967,3 +978,95 @@ def createWithCleanup(cls, testInstance: TestCase, publisher: LogPublisher) -> S
         publisher.addObserver(obs)
         testInstance.addCleanup(lambda: publisher.removeObserver(obs))
         return obs
+
+
+_T = TypeVar("_T")
+
+
+def benchmarkWithReactor(
```

Comment:
Should this be public?

If so, I think it's a `.feature` newsfragment, rather than a `.misc`.

Reply:
Maybe should be private, not really intended for use outside of Twisted.

Reply:
This should be private... I forgot that twisted.internet.testing is for public helpers.

We might want to create `twisted.internet._testing` for all our private stuff

Reply:
I'd rename it `_benchmarkWithReactor` and call it good for now.